Gemfile.lock

```diff
@@ -224,6 +231,8 @@ GEM
       letter_opener (~> 1.0)
       railties (>= 3.2)
     link_header (0.0.8)
+    local_time (1.0.3)
+      coffee-rails
```

Comment:
There are some libs that use jquery and dont need coffee that do "time ago" and some that do "replace utc time with local tz time" ... but this was the best I could find that did both, and it pulls in coffee-rails.

If "lets not have coffeescript anywhere" is a higher level goal, I can revisit and either use two separate libs, or look for another that does both with jquery-only.